cif_core.dic

```diff
@@ -22065,6 +22210,54 @@ save_atom_site_aniso.matrix_b_su
 
 save_
 
+save_atom_site_aniso.matrix_beta
```

Comment:
Ok, so I see that @rowlesmr came up with an alternative solution -- to merge the two dREL functions into one. Seems sound!

Since `_atom_site_aniso.matrix_beta` and `_atom_site.tensor_beta` virtually described the same data item, I merged the two definitions (instead of one of them being declared as deprecated).

One remaining thing is to update the description of `_atom_site_aniso.matrix_beta` so that the mathematical notation in formulae matches the one used in `aniso_betaij` save frame from the `templ_attr.cif`.

@rowlesmr , would you have the time to look into that (given that you are probably best familiar with the notation in `aniso_betaij`)?

Reply:
Thanks!